exec/kubernetes_exec_manager.go

```diff
@@ -109,7 +111,7 @@ func (manager *KubernetesExecManager) Create(machineExec *model.MachineExec) (in
 	for _, c := range containersInfo {
 		containers = append(containers, c.PodName+"\\"+c.ContainerName)
 	}
-	return -1, fmt.Errorf("failed to initialize terminal in any of {%s}", strings.Join(containers, ", "))
+	return -1, fmt.Errorf("failed to initialize terminal in any of {%s} -- errors: %v", strings.Join(containers, ", "), errors)
```

Comment:
Does it make sense to groups errors by containers, I mean:
```
falied to initialize terminal in any of containers. errors:
- che-machine-exec: `sh command is not available`
- dev: `the workspace SA does not have permissions to create exec in pod ...`
```

Reply:
Updated, great idea.